### ISSUE TITLE
DCS server saved games path changed (again)?

### ISSUE THREAD

**Issue by CC1337** [Aterfax/DCS-World-Dedicated-Server-Docker#87]
Hi,

I am setting this up the very first time, means I have no comparison from earlier installations.
Great project!

I had the violation errors as described in readme ALWAYS but after running initial setup several times and then just trying DCS updater link the server launched (but I have not tested a lot yet).

BUT: the autostart failed because it can't find the path, and according to the paths tried at
https://github.com/Aterfax/DCS-World-Dedicated-Server-Docker/blob/893c6e27e2bace413d5119cf2082e460dda7c6e6/docker/src/helper_functions/find_dcs_dirs_function#L28
either mine is different because of some setup issues, or it got changed.
![grafik](https://github.com/user-attachments/assets/bdb6fef7-a7f5-47e7-a108-8cb5dd425fcf)

when doing a manual modification in the container to test it, autostart works.

**Comment by CC1337** [comment]
Additional info: As a next step in my setup, I have installed the DCSServerBot in the container and enabled this for autostart - this seems to work fine with the paths (bot autostart and server autostart via the bot)

**Comment by rgpollitt** [comment]
> I'll have a look (at the soonest) over the weekend. But if the path has changed again it will prove most annoying.

It has. Same issue here.

**Comment by Aterfax** [comment]
I see this now reflected as:

`/config/.wine/drive_c/users/abc/Saved Games/DCS.dcs_serverrelease/`

**Comment by CC1337** [comment]
Yes, on my container it is exactly `/config/.wine/drive_c/users/abc/Saved Games/DCS.dcs_serverrelease/`
